modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/conversion/swagger/v2_0/SwaggerTypes.java

```diff
@@ -60,9 +65,34 @@ public static String toDefinitionType(SwaggerTypeFormat dataType) {
         } else if ("boolean".equals(dataType.getType())) {
             return "boolean";
         } else {
+            if ("array".equals(dataType.getType())) {
```

Comment:
should array be handled here as it not a primitive? (I don't know, you got the answer :)